mcsema/BC/Lift.cpp

```diff
@@ -219,7 +219,11 @@ static InstTransResult LiftInstIntoBlockImpl(TranslationContext &ctx,
     std::cerr << "Error translating instruction at " << std::hex
               << ctx.natI->get_loc() << "; unsupported opcode " << std::dec
               << opcode << std::endl;
-    return TranslateErrorUnsupported;
+
```

Comment:
So the intent is to turn -ignore-unsupported into a command line option that tries inline-asm if it doesn't know how to be lifted?